packages/@uppy/aws-s3-multipart/src/MultipartUploader.js

```diff
@@ -105,6 +108,12 @@ class MultipartUploader {
           onComplete: this.#onPartComplete(j),
           shouldUseMultipart,
         }
+        if (this.#isRestoring) {
+          const size = i + chunkSize > fileSize ? fileSize - i : chunkSize
```

Comment:
How can the chunk size ever be larger than the file itself?

Reply:
`i + chunkSize` feel free to suggest a more readable version